src/pages/components/modal/accessibility.mdx

```diff
@@ -17,400 +17,109 @@ import {
+
+<Caption>
+  The dialog is resolved by the user pressing Esc or activating a button.
+</Caption>
+
+</Column>
 </Row>
 
-### iOS screen reader tests
+### Focus handling
+
+When the dialog appears, the first item that gets focus depends on the type of
+dialog. Passive dialogs only contain a close button (X), so that takes focus.
+For dialogs which prompt for confirmation or user decision, the primary button
+takes focus (regardless of number of buttons). For destructive interactions, the
+“cancel” button takes focus, not the red danger/delete button. The tab order
+should proceed left and down from whichever item has focus then wrap back to the
+Close button (X).
 
 <Row>
-  <Column noGutterSm>
-    <StructuredListWrapper>
-      <StructuredListRow head>
-        <StructuredListCell head>Environment </StructuredListCell>
-        <StructuredListCell head>Results</StructuredListCell>
-      </StructuredListRow>
-      <StructuredListBody>
-        <StructuredListRow>
-          <StructuredListCell>
-            - iOS version 13.1.3 with VoiceOver
-            <br />
-            - Safari version 13.1.3
-            <br />- Carbon React version 7.7.1
-          </StructuredListCell>
-          <StructuredListCell>
-            <strong>VoiceOver test:</strong>
-            <OrderedList>
-              <ListItem>
-                Swipe left or right, or use the Arrow keys in quick nav mode
-                until you find the Launch modal button.
-              </ListItem>
-              <ListItem>
-                Double tap the button to launch the modal. The close modal
-                button is announces and has focus.
-              </ListItem>
-              <ListItem>
-                Swipe Left or Right to explore the modal. VO announces the text,
-                and focus on the Cancel and Save buttons.
-              </ListItem>
-              <ListItem>
-                After Double tapping on the close or cancel button, the modal
-                closes but focus is stuck on the selected button. Touch the
-                screen anywhere to move out of the focus trap.
-              </ListItem>
-            </OrderedList>
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
+<Column colLg={8}>
+
+![initial focus in the dialog varies across four types of modals](images/modal-accessibility-3.png)
+
+<Caption>The first item with focus varies depending on dialog type.</Caption>
+
+</Column>
 </Row>
+
+## Design recommendations
+
+### Designate the input that takes focus in a transactional
+
+The only time the starting focus would not be on a button is where a dialog
+contains input fields. For such transactional dialogs, designers should annotate
+that the first input field should receive focus. Note that text links are not
+considered a user input in this context.
+
+<DoDontRow>
```

Comment:
Shouldn't the word focus in the image be capitalized? "Focus..."

<img width="779" alt="image" src="https://user-images.githubusercontent.com/11670886/207985728-c8fe69d7-6920-444a-9512-1ef0fe5ac574.png">

Reply:
Also I think the content in the images could be cleaned up and add a little more context like so (my image is just hacked together with a screen shot)

<img width="783" alt="image" src="https://user-images.githubusercontent.com/11670886/207986829-bef9fd48-7ca5-4401-acc1-e7d78bd0ae62.png">

Reply:
@laurenmrice I'll let you tackle these image changes. "Focus" can be capitalized if that's preferred; no idea what you folks use for annotation conventions